tools/BuildAssets/BuildTasks/Microsoft.Azure.Sdk.Build.Tasks/buildTools.props

```diff
@@ -0,0 +1,11 @@
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+	<PropertyGroup>
+		<IsBuildToolsProject>true</IsBuildToolsProject>
+	</PropertyGroup>
+	
+	<PropertyGroup Condition=" '$(IsBuildToolsProject)' == 'true' ">
```

Comment:
Why condition this here? I don't see any way that IsBuildToolsProject isn't true from this spot.

Reply:
@weshaggard there are often times when the build scripts evolve and properties get rearranged.
This is just to keep clear that this property group is applicable only under the certain condition.
So in case ever the property group above (line 3) ever get's into any other file, having the condition check right now will save me lot of headache, if I ever forget to add that condition back in.

Reply:
OK that seems a little overkill to me but I don't have an issue with it. I would have just put both of them in the same property group to indicate that they are connected.